.circleci/config.yml

```diff
@@ -6,17 +6,69 @@ jobs:
 
   build-python36:
     docker:
-      - image: ocrd/core
+      - image: ubuntu:18.04
     steps:
-      - run: apt-get update && apt-get install -y --no-install-recommends make git curl
+      - run: apt-get update
+      - run: apt-get install -y software-properties-common
+      - run: add-apt-repository -y ppa:deadsnakes/ppa
```

Comment:
Why `ppa:deadsnakes/ppa`, Python 3.9 perhaps?

Reply:
Yes, this offers a clean selection of all minor python versions, to make it easy to test all versions.